### PR TITLE
feat: added `Recent History` UI on the Stake page

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* feat: added `Recent History` UI on the Stake page

**Check list**
- [ ] contains breaking changes
- [x] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**This pull request makes the following changes:**

**Adds**
* feat: added `Recent History` UI on the Stake page
image

### CODE REVIEW

**Review by bobo-k2 — APPROVED** [review]
LGTM

**Review by Kahonnohak — APPROVED** [review]
Great work! Love it!

File: src/modules/information/recent-history/stake/index.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  dapps: DappCombinedInfo[];
+  subScan: string;
+}): Promise<RecentHistory[]> => {
+  const url = `${TOKEN_API_URL}/v1/${network}/dapps-staking/stats/user/${address}/1%20years`;
+  const result = await axios.get<UserStakeHistory[]>(url);
+  const numberOfHistories = 5;
+  return result.data
+    .filter(
+      (it) =>
+        it.transaction === 'BondAndStake' ||
```
Comment (sirius651) [inline src/modules/information/recent-history/stake/index.ts:48]: can we use separate type value instead of hardcoded string?

Comment (impelcrypto) [reply src/modules/information/recent-history/stake/index.ts:48]: ok, I've defined type annotation in [this](https://github.com/AstarNetwork/astar-apps/pull/602/commits/cca91411a2dbee35ba79a4e5a3ca054d842da6c9) commit.

**Pull request merged**